.github/workflows/run-tests.yml

```diff
@@ -21,10 +21,7 @@ jobs:
         uses: actions/checkout@v3
       - name: Set up Node.js
         uses: actions/setup-node@v3
-        with:
```

Comment:
Are you supposed to have these changes for this ticket?

Reply:
Since the endpoint I was testing was being changed in dev, I had to pull again to get all those changes and I guess they are being shown as changes I made. I didn't write or change anything other than the test I wrote

Reply:
Anthony also went in and changed some stuff when he was helping me figure out the errors

Reply:
Can you pull from develop again and see if these changes go away?